c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -1818,7 +1823,9 @@ static void data_out_speed_up_command(sdp_msg_pure_data *msg) {
                     data_out_transaction_id, message->transaction_id);
             return;
         }
+#if 0
```

Comment:
Why do if #0
Would it not be better to either of
1. Just comment out
2. And if def with a flag that could be set at compile time
3. Just delete they easy enough to add back in as as when required.

Reply:
Agreed, but will leave for a later PR to do this properly with log_debug etc.